src/client/chat.js

```diff
@@ -106,7 +113,8 @@ module.exports = function (client, options) {
     const expired = !packet.timestamp || tsDelta > messageExpireTime || tsDelta < 0
     const verified = updateAndValidateChat(packet.senderUuid, packet.previousSignature, packet.signature, hash.digest()) && !expired
     client.emit('playerChat', {
-      message: packet.plainMessage || packet.unsignedChatContent,
+      plainMessage: packet.plainMessage,
```

Comment:
Can you change the documentation to reflect that change ?

Reply:
Done in latest commit

Reply:
The point of the event is to abstract internals like this, why separate this to two fields?

Reply:
Because both are needed to properly format the chat message

Reply:
Can you explain?

Reply:
The chat example here is also broken with this change

Reply:
Where?

Reply:
examples/chat

Reply:
> Can you explain?

Passing them the previous way means we will have to check whether `message` is a plaintext string or stringified JSON.

This not only adds unnecessary complications, but also fails to account for the rare case where a user might send a message that is also valid JSON, in which case it will be parsed incorrectly.

Reply:
Which is which? Can you add this explanation to the doc and handling to the examples?

Reply:
> Which is which? Can you add this explanation to the doc and handling to the examples?

unsignedContent is stringified JSON. plainMessage is plain text

Docs should already be updated.

Going to update examples in a new PR

Reply:
Don't see this explained in the doc, this is important for users listening to chat to know. To fix the examples you can also revert the `message` -> `plainMessage` change and keep the new unsigned field

Reply:
It is there. https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/docs/API.md#playerchat-event